src/kvstore/kvstore_dist_server.h

```diff
@@ -286,13 +286,14 @@ class KVStoreDistServer {
           // instead of calling BinaryComputeRspRsp directly
           using namespace mshadow;
           Engine::Get()->PushSync([recved, merged, out](RunContext ctx) {
-              std::vector<NDArray> inputs, outputs;
-              inputs.push_back(recved);
-              inputs.push_back(merged.array);
-              outputs.push_back(out);
-              op::BinaryComputeRspRspImpl<cpu, cpu>({}, {}, inputs, {kWriteTo}, outputs);
-            }, recved.ctx(), const_vars, {out.var()},
-            FnProperty::kNormal, 0, PROFILER_MESSAGE_FUNCNAME);
+                                    std::vector<NDArray> inputs, outputs;
```

Comment:
I'd prefer not to make the internal function public. I don't think there's any advantage to calling it directly.